Content.Server/GameObjects/EntitySystems/HandsSystem.cs

```diff
@@ -166,8 +166,8 @@ private bool HandleThrowItem(ICommonSession session, GridCoordinates coords, Ent
             if (!handsComp.ThrowItem())
                 return false;
 
-            // pop off an item, or throw the single item in hand.
-            if (!throwEnt.TryGetComponent(out StackComponent stackComp) || stackComp.Count < 2)
+            // throw the item, split off from a stack if it's meant to be thrown individually
+            if (!throwEnt.TryGetComponent(out StackComponent stackComp) || stackComp.Count < 2 || !stackComp.ThrowIndividually)
```

Comment:
put your check first - it's a simple property check, compared to TryGetComponent. Should help speed up the process

Reply:
My check depends on the output of TryGetComponent, as far as I'm aware it won't work without it.

Reply:
@AndriiYukhymchak 
![image](https://user-images.githubusercontent.com/6979049/74268671-40143580-4cd6-11ea-999e-edb431fc03ba.png)

Reply:
@AndriiYukhymchak ![image](https://user-images.githubusercontent.com/33945568/74268722-56ba8c80-4cd6-11ea-8fa7-49a0fb776d9c.png)

Reply:
Ahahaha... imagine suddenly realizing that you read the code like idiot